3-enrich/stream-enrich/nsq/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sources/NsqSource.scala

```diff
@@ -90,14 +87,14 @@ class NsqSource private (
 
     val errorCallback = new NSQErrorCallback {
       override def error(e: NSQException): Unit =
-        log.error(s"Exception while consuming topic $topicName", e)
+        log.error(s"Exception while consuming topic $${config.in.raw}", e)
```

Comment:
Is this `$` redundant?

Reply:
It is there to escape $ which would cause an error. The s" is needed to silence a warning.